CardsCollectionViewLayout.swift

```diff
@@ -59,6 +59,11 @@ open class CardsCollectionViewLayout: UICollectionViewLayout {
 
     let percentageDeltaOffset = CGFloat(deltaOffset) / collectionView.bounds.width
 
+    // ToDO: fix crash
+    if minVisibleIndex > totalItemsCount {
+        minVisibleIndex = totalItemsCount
+    }
```

Comment:
Do you know why `minVisibleIndex` could be larger than `totalItemsCount` ? Does this happen when `collectionView.contentOffset.x` is too large? Could it be that the real root cause is in the different place?